app/models/annotation/Annotation.scala

```diff
@@ -234,14 +234,12 @@ object AnnotationDAO extends SecuredBaseDAO[Annotation]
     find(Json.obj(
       "_task" -> _task,
       "typ" -> annotationType,
-      "$or" -> Json.arr(
-        Json.obj("state.isAssigned" -> true),
-        Json.obj("state.isFinished" -> true))))
+      "$or" -> Json.obj("state" -> Finished)))
```

Comment:
Is this correct or should it just be `"state" -> Finished` ?

Reply:
I think in mongo queries this is a prefix-or, meaning *either* isAssigned->true *or* isFinished->true.
That means it can now be omitted if you just have one condition.

Reply:
Ok now I understand the difference between "or" and "in" (and why "or" was needed here).
I fixed it now.